src/views/GovernanceOverview/__snapshots__/index.test.tsx.snap

```diff
@@ -255,14 +255,10 @@ exports[`The governance overview page matches the snapshot 1`] = `
             className="fa fa-angle-right margin-x-05"
           />
         </li>
-        <li>
-          <a
-            aria-current="location"
-            href="/governance-overview"
-            onClick={[Function]}
-          >
-            Add a new system or service
-          </a>
+        <li
```

Comment:
So since this has `aria-current`, this is the last "crumb" in the breadcrumb, which means it's the current page. It shouldn't link anywhere. If you look at the previous diff, it has `href="/governance-overview"`. That's pointing to itself. I don't think that's necessary.